.vscode/settings.json

```
@@ -23,5 +23,5 @@
         "**/*.pyc": true,
         "**/env": true
     },
-    "python.pythonPath": "venv/bin/python",
+    "python.pythonPath": "/usr/bin/python3",
```

Si ponemos como interprete /usr/bin/python3, se asumo que todos los paquetes estan instalados globalmente. Es mejor que el interprete sea env/bin/python ya que es el mismo del virtual environment

```suggestion
    "python.pythonPath": "env/bin/python",
```